### ISSUE TITLE
command "befores" running in parallel or series

### ISSUE THREAD

**Issue by scottcorgan** [scottcorgan/nash#16]
It would be nice to have the `befores` run in series or parallel
### Proposed:

**Series**

``` js
cli.command('somecommand')
  .before(function () {})
  .before(function () {})
```

**Parallel**

``` js
cli.command('somecommand')
  .before([function () {}, function () {}, function () {}])
```

--or--

``` js
cli.command('somecommand')
  .before(function () {}, function () {}, function () {})
```
